lib/fortnox/api/models/row/entity.rb

```diff
@@ -0,0 +1,66 @@
+require "fortnox/api/models/entity/base"
+
+module Fortnox
+  module API
+    module Entities
+      class Row < Fortnox::API::Entities::Base
```

Comment:
Does it need to inherit from `Base`? If it's only a value object it shouldn't have to. Look at it as a larger version of `Currency` or `CountryCode` ...